cli/internal/runsummary/vercel.go

```diff
@@ -0,0 +1,45 @@
+package runsummary
+
+type vercelRunResponse struct {
+	ID string
+}
+
+type vercelRunPayload struct {
```

Comment:
Where does this come from? Are we hand-writing these datastructures?

Reply:
yeah pretty much... I can ping you a doc in slack